files/Aboot/boot0.j2

```diff
@@ -41,11 +41,13 @@ parse_environment_config() {
 }
 
 extract_image() {
-    ## Clean old directory for read-write layer
-    rm -rf "$target_path/rw"
-
-    ## Clean docker directory
-    rm -rf "$target_path/{{ DOCKERFS_DIR }}"
+    ## Remove all the other unnecssary files except swi file, boot-config and boot0
+    for f in `ls -A $target_path`; do
+        if [ $f != "${swipath##*/}" ] && [ $f != "boot-config" ] && [ $f != "boot0" ]; then
```

Comment:
Ah. If so, we don't need to preserve it here. Was thinking it locates in /mnt/flash lol.